config/routes.rb

```diff
@@ -1,4 +1,7 @@
 Rails.application.routes.draw do
-  root 'home#index'
-  resources :cards
+  root 'reviews#new'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.